lib/parsers/document_parser.js

```diff
@@ -305,6 +303,26 @@ function fillAssociations() {
   }
 }
 
+function convertAnnotationsToOptions(annotations) {
+  const result = {};
+  annotations.forEach(o => {
+    const value = o.method ? o.method : true;
```

Comment:
you can use here `o.method || true`
by the way, why true?

Reply:
well annotations have a key and a value, some annotations do not need a value ex: `@id`, you would just check `if (annotation.method)` which would evaluate to `true`.